tests/unittests/Foundation/TestUtils.h

```diff
@@ -15,6 +15,17 @@
 //******************************************************************************
 #pragma once
 
-#import <Foundation/NSOrderedSet.h>
+#import <Foundation/Foundation.h>
+#import <functional>
+#include <memory>
 
-void assertOrderedSetContent(NSOrderedSet* set, NSObject* first, ...);
\ No newline at end of file
+#define _CONCAT(x, y) x##y
+#define CONCAT(x, y) _CONCAT(x, y)
+
+#define _SCOPE_GUARD(STATEMENT) std::unique_ptr<void, std::function<void(void*)>> CONCAT(_closeScope_, __LINE__)((void*)0x1, STATEMENT)
+
+void assertOrderedSetContent(NSOrderedSet* set, NSObject* first, ...);
+NSString* getModulePath();
```

Comment:
Do things still build on Windows like this? (I feel like I wanted to do this earlier but didn't for some reason)